### ISSUE TITLE
Implements source running for each languages by script and run testcase

### ISSUE THREAD

**Issue by moreal** [boj-vs-code/client#3]

**Comment by moreal** [comment]
Can I support too many languages?

**Comment by moreal** [comment]
We can support it with wsl.
And I'll provide languages (only c, cpp, python), which implements `ILanguage`,

**Comment by moreal** [comment]
```typescript
interface ILanguage {
  function install(): void;
  function run(string): string;
  function test(string, string): bool;
};
```

**Comment by moreal** [comment]
This is resolved by #44 and #46.
